pkg/controller/persistence/dgraph/dgraph.go

```diff
@@ -85,6 +86,7 @@ func CreateSchema() error {
 }
 
 // GetUID returns the UID of the node in the Dgraph
+// returns empty string if error has occured
 func GetUID(dg *dgo.Dgraph, id string, nodeType string) string {
```

Comment:
Verify the scope of this function and accordingly update the case for this function?